### DISCUSSION TITLE
android

### DISCUSSION THREAD

**Discussion by icegood** [TresysTechnology/selint#253] [Q&A] [answered]
is it possible to adapt tool for android? If not, what is preferrable instead?

**Comment by dburgener** [comment, marked as answer]
There are a few issues with running SELint on Android.  See some discussion here: https://github.com/SELinuxProject/selint/issues/199

Getting selint to run on Android out of the box isn't a priority for me to work on, but I would happily accept patches to make it happen.

The current set of checks are heavily refpolicy style focused and many of them may be of less use on Android.

The other SELinux linter I'm aware of is James Carter's selpoltools, and you might want to check that out: https://github.com/jwcart2/selpoltools.  Based on its docs, it is also refpolicy focused, and I'm not aware of its status regarding Android policy support.
